tests/test.py

```diff
@@ -35,7 +35,6 @@
     import tests.test
 
     tests.test.main()
-    sys.exit(0)
```

Comment:
@jonasBoss was using the sys.exit here to prevent importing this file twice or something

Reply:
since we have the sys.exit at the end of main() this is just fine. we don't need it here, I just added it because it was missing before. Reporting if the test passed makes totally sense.